### PR TITLE
INS-27788 - Allow custom SAN specification at cluster creation

### PR DESCRIPTION
**Pull request by nikunj-netapp**
Adding support to allow customers to specify custom Subject Alternate Names at cluster creation for Postgres, Apache Kafka and managed Apache Zookeeper

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,13 @@
 BIN_NAME=terraform-provider-instaclustr
 
-VERSION=2.0.99
+VERSION=2.0.100
 
 FULL_BIN_NAME="${BIN_NAME}_v${VERSION}"
 SHASUM_NAME="${BIN_NAME}_${VERSION}_SHA256SUMS"
+GOOS="darwin"
```
Comment (nikunj-netapp) [inline Makefile:7]: Making these into variables to they can be specified as required by the os and arch while running make install

**Review by ankit-agrawal26 — APPROVED** [review]

File: docs/data-sources/kafka_cluster_v2_instance.md
Comment (nikunj-netapp) [inline docs/data-sources/kafka_cluster_v2_instance.md:1]: my changes are on line 194-196

File: docs/data-sources/opensearch_cluster_v2_instance.md
Comment (nikunj-netapp) [inline docs/data-sources/opensearch_cluster_v2_instance.md:1]: changes from other PRs

File: docs/data-sources/postgresql_cluster_v2_instance.md
Comment (nikunj-netapp) [inline docs/data-sources/postgresql_cluster_v2_instance.md:1]: changes from app branch INS-27788 are on 122-124

File: docs/data-sources/zookeeper_cluster_v2_instance.md
Comment (nikunj-netapp) [inline docs/data-sources/zookeeper_cluster_v2_instance.md:1]: changes from app branch INS-27788 are from 104-106

File: docs/index.md
Comment (nikunj-netapp) [inline docs/index.md:1]: changes from other PRs

File: docs/resources/cassandra_cluster_v2.md
Comment (nikunj-netapp) [inline docs/resources/cassandra_cluster_v2.md:1]: changes from other PRs

File: docs/resources/kafka_cluster_v2.md
Comment (nikunj-netapp) [inline docs/resources/kafka_cluster_v2.md:1]: changes from app branch INS-27788 are on 215-217

File: docs/resources/opensearch_cluster_v2.md
Comment (nikunj-netapp) [inline docs/resources/opensearch_cluster_v2.md:1]: changes from other PRs

File: docs/resources/postgresql_cluster_v2.md
Comment (nikunj-netapp) [inline docs/resources/postgresql_cluster_v2.md:1]: changes from app branch INS-27788 are on 138-140

File: docs/resources/zookeeper_cluster_v2.md
Comment (nikunj-netapp) [inline docs/resources/zookeeper_cluster_v2.md:1]: changes from app branch INS-27788 are on 114-116

**Review by iclonghoang — APPROVED** [review]

**Comment by iclonghoang** [comment]
thanks for fixing docs-base file that customer platform team missed

**Review by yuzhouchen-instaclustr — APPROVED** [review]

**Review by jacobhuanginstaclustr — APPROVED** [review]

**Pull request merged**